backend/lib/services/members/MemberManager.js

```diff
@@ -162,13 +175,33 @@ class MemberManager {
   async deleteServiceAccount (item) {
     const { namespace, name } = Member.parseUsername(item.id)
     if (namespace !== this.namespace) {
-      return
+      return // do not throw an error as it is allowed to add foreign service accounts
```

Comment:
is it really allowed to delete a service account from another namespace?
if yes, the comment needs to be adapted anyhow: `add` -> `delete`

Reply:
renamed comment to make it clearer